### ISSUE TITLE
Issue with overwriting Messages Priority

### ISSUE THREAD

**Issue by dwolbeck** [ubittner/SymconPushover#4]
First of all thank you for your effort!

The setting of message priority via the instance configuration of your module works without problems. If I pass a priority via a script, it will be ignored.

UBPO_SendPushoverNotification($iInstanceID, $sTitle, $sMessage, $sNotificationTone, $iDeliveryPriority, $sDeviceName, $iBildID);

Additionally, I have the question if the parameters "expire" and "retry" of emergency messages can be passed somehow?

Thank you

**Comment by ubittner** [comment]
I will check the request soon.

**Comment by ubittner** [comment]
First I would like to point out that the following function does not exist:
UBPO_SendPushoverNotification($iInstanceID, $sTitle, $sMessage, $sNotificationTone, $iDeliveryPriority, $sDeviceName, $iBildID);

The module uses the following functions (please add the InstanceID at the begining of the parameters for Symcon):
SendPushoverNotification(string $Title, string $Message)
-> This function will get the properties of the configuration form and uses:

SendPushoverNotificationEx(string $Title, string $Message, string $Sound, int $Priority, string $Device)
-> So this function is flexible for parameters without the configuration form of Symcon.

In both functions is a priotity parameter which will be passed are static:
 $retry = '30';
 $expire = '3600';
So here you don't have a chance (at the moment to use your own values), but I can improve that in a future release.

And last but not least:
SendPushoverImageAttachmentEx(string $Title, string $Message, string $Sound, int $Priority, string $Device, int $ImageID)

So please tell me your exact script string, that I can verify what is maybe going wrong.

Regards 

Uli

**Comment by dwolbeck** [comment]
Hello, Uli, 

Thank you very much for your quick answer and the effort. I didn't read your PHP examples carefully enough and mistakenly didn't see that they were multiple functions. I had only adopted the extended parameters.

I have now used the function "SendPushoverNotificationEx" instead of "SendPushoverNotification" and it works great!

If the priority parameters "retry" and "expire" could also be configured in a future version, this would be great.

Once again many thanks and many greetings
Dirk
